dist/index.js

```diff
@@ -0,0 +1,14 @@
+import { TabNavigator } from 'react-navigation';
```

Comment:
Don't include the dist build in your dev commits. When you run a build to regenerate the dist files, do it as a separate commit called something like: `build: update`.